src/components/attendees/AttendeeTable.tsx

```diff
@@ -35,68 +51,170 @@ export default function AttendeeTable() {
+								leave="ease-in duration-200"
+								leaveFrom="opacity-100 translate-y-0 sm:scale-100"
+								leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+							>
+								<Dialog.Panel className="relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all w-5/6 pb-10 max-w-[900px]">
+									<div className="w-full h-max pb-10 bg-white rounded-xl flex flex-col">
+										<div className="px-8 py-4 text-xl font-bold flex items-start justify-start gap-4">
+											<button
+												className="h-8 w-8 bg-gray-200 rounded-full flex items-center justify-center text-attendee-name"
+												onClick={() => {
+													setPopUp((popUp) => ({
+														...popUp,
+														open: false,
+													}));
+												}}
+											>
+												<CloseIcon />
+											</button>
+											<div className=" text-popup-heading text-bold text-xl">
+												{`${popUp.selectedAttendee.firstName} ${popUp.selectedAttendee.lastName}`}
+											</div>
+										</div>
+										<AttendeePopUp
+											selectedAttendee={
+												popUp.selectedAttendee
+											}
+										/>
+									</div>
+								</Dialog.Panel>
+							</Transition.Child>
+						</div>
+					</div>
+				</Dialog>
+			</Transition.Root>
+			{/* Search Component */}
+			<div className="h-auto w-full px-5">
+				<h1 className=" font-bold text-3xl text-subHeading-color mb-4">
+					Attendees
+				</h1>
+				<input
+					id="search"
+					className="mb-8"
+					type={'text'}
+					placeholder="Search Attendees"
+					onChange={(e) => {
+						setSearch(e.target.value);
+					}}
+				/>
+			</div>
+			{/* Table Component */}
+			<div className="h-full w-full overflow-auto  p-5">
+				<table className="w-full border-collapse border-spacing-0 p-5">
+					<tbody>
+						<tr>
+							<th>
+								<div id="nameContent">
+									Name
+									{!namesAscending ? (
+										<button>
+											<SortIcon id="ascendingSort" />
+										</button>
+									) : (
+										<button>
+											<SortIcon id="descendingSort" />
+										</button>
+									)}
+								</div>
+							</th>
+							<th> Email </th>
+							<th> Discord </th>
+							<th> Auth </th>
+							<th> School </th>
+							<th> Accepted </th>
+							<th> Confirmed </th>
+							<th> Checked In </th>
+							<th id="filterIcon" className="">
+								<div className=" flex w-full justify-end align-center">
+									<FilterIcon />
+								</div>
+							</th>
+						</tr>
+						{attendeeData.map((attendee) => {
+							return (
+								<tr key={attendee.id}>
+									<AttendeeRow
+										key={attendee.id}
+										attendee={attendee}
+									/>
+									<td>
+										<button
+											id="openUser"
+											onClick={() => {
+												if (!popUp.seenAttendeePopUp) {
+													setPopUp((popUp) => ({
+														...popUp,
+														seenAttendeePopUp: true,
```

Comment:
It's unclear what this property is for as it doesn't seem to be used right now. Even if it is, running code in the else seems to not do anything useful.

If you want to update state for some reason (which it should do right now in either case since you're providing a new object), you could call:
```
setPopUp((popUp) => ({
    //...popUp, this doesn't do anything if you're setting all three properties anyways
    seenAttendeePopUp: true,
    selectedAttendee: attendee,
    open: true,
}));
```
This removes the need for a conditional but still updates state.